keras_preprocessing/image.py

```diff
@@ -697,6 +697,9 @@ def __init__(self,
                  preprocessing_function=None,
                  data_format=None,
                  validation_split=0.0):
+        if brightness_range is not None and len(brightness_range) != 2:
```

Comment:
Please move this block *after* the block of assignments to be compliant with the style of this constructor. Around line 725.